modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -8,6 +8,17 @@
 		"issueKey": "LPS-184731",
 		"to": "addFolder(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#)"
 	},
+	{
+		"from": "themeDisplay.getPortletGroupId",
+		"issueKey": "LPS-186809",
+		"to": "themeDisplay.getScopeGroupId",
```

Comment:
@TamyrisTorres In this case we have to test the type if it's `themeDisplay` when possible. So should have the `classNames` attribute.